test/fixtures/linter/rules/NoDeprecatedApi/Configuration.beforeBootstrap.qunit.js

```diff
@@ -0,0 +1,10 @@
+// This legacy file was a False Positive for the new Test Starter detections.
```

Comment:
It's not a legacy file. It is part of the Configuration test in the core library. Maybe add a reference to the original location in openui5, or we come up with a minimal example where the same problem can be reproduced.